2-Git/README.md

```diff
@@ -0,0 +1,297 @@
+   ```
+   ![Screenshot_21](screenshots/Screenshot_21.png)
+
+  ## 5. Create Pull Request from branch "hw_git" to the master branch and assign me as reviewer to this merge request
+
+   ![Screenshot_22](screenshots/Screenshot_22.png)
+
+# 3
+  ## 1. Set up Gitlab CE in docker container
+    
+   ```
+   $ docker run -d -p 80 -p 22 gitlab/gitlab-ce:latest
+   ```
+
+   ![Screenshot_23](screenshots/Screenshot_23.png)
+
+   Configure port forwarding to host machine ports 9080 and 9022 to have access in order to WebUI from host machine's browser: (localhost:9080):
+
+   ![Screenshot_24](screenshots/Screenshot_24.png)
+
+  ## 2. Log in as root
+    
+   ![Screenshot_25](screenshots/Screenshot_25.png)
+   ![Screenshot_26](screenshots/Screenshot_26.png)
+   ![Screenshot_27](screenshots/Screenshot_27.png)
+    
+  ## 3. Create group "hw_git"
+    
+   ![Screenshot_28](screenshots/Screenshot_28.png)
+   ![Screenshot_29](screenshots/Screenshot_29.png)
+   ![Screenshot_30](screenshots/Screenshot_30.png)
+
+  ## 4. Create two users: maintainer and developer
+    
+   ![Screenshot_31](screenshots/Screenshot_31.png)
+   ![Screenshot_32](screenshots/Screenshot_32.png)
+   ![Screenshot_33](screenshots/Screenshot_33.png)
+   ![Screenshot_34](screenshots/Screenshot_34.png)
+   ![Screenshot_35](screenshots/Screenshot_35.png)
+   ![Screenshot_36](screenshots/Screenshot_36.png)
+   ![Screenshot_37](screenshots/Screenshot_37.png)
+
+  ## 5. Add these users to the group and set them proper permissions in the group (maintainer – maintainer, developer – developer)
+    
+   ![Screenshot_38](screenshots/Screenshot_38.png)
+   ![Screenshot_39](screenshots/Screenshot_39.png)
+   ![Screenshot_40](screenshots/Screenshot_40.png)
+   ![Screenshot_41](screenshots/Screenshot_41.png)
+   ![Screenshot_42](screenshots/Screenshot_42.png)
+
+  ## 6. Create new project with any name
+    
+   ![Screenshot_43](screenshots/Screenshot_43.png)
+   ![Screenshot_44](screenshots/Screenshot_44.png)
+   ![Screenshot_45](screenshots/Screenshot_45.png)
+
+
+  ## 7. Create all branches for GitFlow in this project (you can create one feature and one release branch and don't create hotfix branch)
+    
+   ![Screenshot_46](screenshots/Screenshot_46.png)
+   ![Screenshot_47](screenshots/Screenshot_47.png)
+   ![Screenshot_48](screenshots/Screenshot_48.png)
+   ![Screenshot_49](screenshots/Screenshot_49.png)
+   ![Screenshot_50](screenshots/Screenshot_50.png)
+   ![Screenshot_51](screenshots/Screenshot_51.png)
+   ![Screenshot_52](screenshots/Screenshot_52.png)
+
+  ## 8. Protect master branch to allow only maintainers to merge into it, and restrict all to push there
+    
+   ![Screenshot_53](screenshots/Screenshot_53.png)
+   ![Screenshot_54](screenshots/Screenshot_54.png)
+   ![Screenshot_55](screenshots/Screenshot_55.png)
+   ![Screenshot_56](screenshots/Screenshot_56.png)
+
+  ## 9. Protect release branches by wildcard (release-* for example) and allow only maintainers to merge into it

```

Comment:
The `release*` is not the same as `release-*`. 

No fixes required, but next time, please be more attentive.

Reply:
Okay, I just thought that this is the place where we can choose wildcards by ourselves :)